content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
 complexes but may be less effective since on average protein complexes have
 fewer sequence homologs.
 
+### Protein-Protein Interactions
+
+Protein-protein interactions (PPIs) are highly specific and non-accidental physical contacts between proteins which occur for purposes other than generic protein production or degradation [@doi:10.1371/journal.pcbi.1000807].
+PPIs are key to many cellular processes like metabolism and immune responses.
+Abundant interaction data have been generated in-part thanks to advances in high-throughput screening methods, such as yeast two-hybrid and affinity-purification with mass spectrometry.
+However, because many PPIs are transient or dependent on biological context, high-throughput methods can fail to capture a number of interactions.
```

Comment:
This sentence alone might be enough to motivate the need for PPI prediction.  Then you could cut the line about false positive rates, because a reader might wonder whether computational predictions really have lower false positive rates than Y2H.